controls/radrichtextbox/features/merge-documents/append.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Append Documents
+page_title: Append Documents
+description: Append Documents
+slug: radrichtextbox-features-merge-append-documents
+tags: raddocumentmerger, merge, insert, documents
+published: True
+---
+
+# Append Documents
+
+The [RadDocumentMerger]({%slug radrichtextbox-features-merge-raddocumentmerger%}) class exposes the AppendDocument() method that enables you to append a [RadDocument]({%slug radrichtextbox-features-document-elements-raddocument%}) instances.
+
+Along with the RadDocument instance, the overloads of the AppendDocument() method allow you to pass a parameter of type **AppendDocumentOptions**. You can use this parameter to customize the way the merge conflicts are resolved and the new document is inserted. The available properties are as follows:
+
+* **ConflictingStylesResolutionMode**: Determines the mode of resolving conflicts between styles with same IDs. This property is of type ConflictingStylesResolutionMode and could have the following enum values:
+
+	* **UseTargetStyle**: The style of the target document is used. E.g. if the target and source documents have a style with a name "MyStyle", but with different properties, the one from the target is used by the merged document and the one from the source document is not added to the style repository of the target document.
+	
+	* **RenameSourceStyle**: The style of the source document is renamed and used. E.g. if the target and source documents have a style with a name "MyStyle", but with different properties, the one from the source is renamed to "MyStyle\_1", added to the style repository of the target document and used by the merged from the source document elements. *This is the default value*.
+
+* **FirstSourceSectionPropertiesResolutionMode**: Determines whether the source document should be inserted in the target with a preceding SectionBreak, should the source section properties be omitted or preserved. **The value is used only for the first section of the source document**. The rest of the section elements and their properties remain unchanged. 
+
+	The property is of type **SectionPropertiesResolutionMode** and could have the following enum values:
+
+	* **NoSectionBreak**: Append the document elements from the source RadDocument into the target without a preceding section break. Only the block elements from the first source section are inserted. The first section properties, headers and footers are omitted. 
+	
+	* **Source**: Append the first source section as it is, with a preceding section break. The type of the break is specified in the section properties of the first source section. The rest of the section properties, headers and footers are preserved. E.g. if the source section has a *landscape* orientation and the target has *portrait*, the result section will be with *landscape* orientation.
```

Comment:
Why "Landscape" starts with capital letter, but "portrait" doesn't? I think that both have the same context - they are orientation types and should both start with capital or small letter.